cli/connhelper/connhelper.go

```diff
@@ -53,6 +53,16 @@ func GetConnectionHelper(daemonURL string) (*ConnectionHelper, error) {
 	return nil, err
 }
 
+// GetCommandConnectionHelper returns a ConnectionHelp constructed from an arbitrary command.
+func GetCommandConnectionHelper(cmd string, flags ...string) (*ConnectionHelper, error) {
+	return &ConnectionHelper{
+		Dialer: func(ctx context.Context, network, addr string) (net.Conn, error) {
+			return newCommandConn(ctx, cmd, flags...)
+		},
+		Host: "http://docker",
```

Comment:
Also thinking out loud while trying to grasp this PR; we should add comments with these (future readers may be confused and think "what's this magic `docker` host?"

Reply:
This came pretty much verbatim from the existing example, I don't know where it came from now really where it shows up...